app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -1,17 +1,32 @@
 {{#if this.modelTaskInstance.isRunning}}
     <LoadingIndicator data-test-loading-indicator @dark={{true}} />
 {{else}}
-    <PowerSelect
-        @options={{this.departments}}
-        @searchEnabled={{true}}
-        @selected={{this.department}}
-        @triggerClass={{local-class 'select'}}
-        @search={{perform this.searchDepartment}}
-        @onChange={{action this.onSelectChange}}
-        as |department|
-    >
-        {{department}}
-    </PowerSelect>
+    <div local-class='table-filters'>
+        <div local-class='filter-controls'>
+            <div local-class='orcid-switch'>
+                <label for='switches-container' local-class='orcid-toggle-label'>{{t 'institutions.dashboard.users_list.has_orcid'}}</label>
+                <label local-class='switch'>
+                    <input type='checkbox' checked={{this.hasOrcid}} {{on 'change' (fn this.toggleOrcidFilter (not this.hasOrcid))}}>
```

Comment:
The input doesn't have a valid label right now, so you will have to add `id='switches-container'` to the `<input>` element to associate the "Has ORCID" label to this input